### PR TITLE
feat: Accordion and AccordionItem

### PR DESCRIPTION
**Pull request by HaakonSvane**
- Add Accordion component in mad components
- Add AccordionScreen to chronicles

### CODE REVIEW

File: packages/components/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    isFirstItem: true,
+    isLastItem: true,
+});
+
+export type AccordionProps = {
+    singular?: boolean;
+}
+
+export const Accordion = ({
+    singular = false,
```
Comment (hathibelagal-dev2) [inline packages/components/src/components/Accordion/Accordion.tsx:21]: singular is never used

File: packages/components/src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    children,
+    ...rest
+}: React.PropsWithChildren<AccordionProps & ViewProps>) => {
```
Comment (hathibelagal-dev2) [inline packages/components/src/components/Accordion/Accordion.tsx:24]: Is React.PropsWithChildren necessary when you are adding ViewProps?

**Review by hathibelagal-dev2 — APPROVED** [review]
Looks great!

File: packages/components/src/components/Accordion/Accordion.tsx (hunk 2 above)
Comment (HaakonSvane) [reply packages/components/src/components/Accordion/Accordion.tsx]: Good point 👍 Fixed in c814210

File: packages/components/src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (HaakonSvane) [reply packages/components/src/components/Accordion/Accordion.tsx]: Fixed in c60697c

**Pull request merged**